### PR TITLE
Add access_level service.

### PR DESCRIPTION
**Pull request by Hellowlol**
This service used together with with device tracker can be used to open you charger when you home and require rfid or whitelist when you not.

Using this service will also stop and start the charge when the access is changed.

### CODE REVIEW

**Review by fondberg — APPROVED** [review]

File: custom_components/easee/services.py [hunk 1]
```diff
@@ -285,14 +287,23 @@ async def charger_execute_set_charging_cost(call):
...
             return await function_call(cost_per_kwh, vat, currency)
 
-        _LOGGER.error(
-            "Could not find charger %s", charger_id,
-        )
+        _LOGGER.error("Could not find charger %s", charger_id)
         raise HomeAssistantError("Could not find charger {}".format(charger_id))
 
+    async def charger_execute_set_access(call):
+        """Execute a service to set se cost per kwh for Easee charger site."""
```
Comment (olalid) [inline custom_components/easee/services.py:294]: This comment seems like it is cut&pasted from another function and not relevant here?

**Comment by olalid** [comment]
Great addition @Hellowlol.

**Review by olalid — APPROVED** [review]

File: custom_components/easee/services.py (hunk 1 above)
Comment (Hellowlol) [reply custom_components/easee/services.py]: Your right. Thanks. Its fixed.

**Review by olalid — APPROVED** [review]

**Review by tmjo — APPROVED** [review]

**Comment by fondberg** [comment]
This requires a bump to the easee library. Best done in this PR

**Pull request merged**